ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -54,6 +56,10 @@ class FaultInjector
     LanderExecutionError = 256};
 
 private:
+  float powerTemperatureOverload;
+  
+  void setPowerTemperatureFaultValue(bool b_getTemp);
```

Comment:
Curious about the "b_" in the argument name - have not seen this convention...

Reply:
I can remove and just make them boolgetTemp